src/mscorlib/ref/mscorlib.cs

```diff
@@ -1795,7 +1795,9 @@ public partial class Exception
     [System.Runtime.InteropServices.ComVisibleAttribute(true)]
     public sealed partial class ExecutionEngineException : System.SystemException
     {
-        internal ExecutionEngineException() { }
+        public ExecutionEngineException() { }
+        public ExecutionEngineException(string message) { }
+        public ExecutionEngineException(string message, System.Exception inner) { }
```

Comment:
Isn't the parameter named `innerException`?

https://github.com/dotnet/coreclr/blob/6bd955703e218f334ce941e3bdf5e0fc1aeedc35/src/mscorlib/src/System/ExecutionEngineException.cs#L38